doc/source/.res/code-block/bash/systemd-unit-file.txt

```diff
@@ -0,0 +1,16 @@
+.. code-block:: ini
+   
```

Comment:
Not sure if there is a formatter for the ini type in pygments (most likely there is). If in doubt, you may use guess (code-block:: guess). Anyway, even if you do use an unknown type, the rendering will not be affected.